intrastat_product/__manifest__.py

```diff
@@ -7,11 +7,11 @@
 
 {
     "name": "Intrastat Product",
-    "version": "13.0.1.0.3",
+    "version": "13.0.1.1.0",
```

Comment:
Please next time don't change version number, as it's done by the bot when merging and you avoid possible conflicts.